src/Microsoft.ML.Transforms/PermutationFeatureImportanceExtensions.cs

```diff
@@ -411,69 +408,79 @@ protected MetricStatistics[] InitializeArray(int length)
     }
 
     /// <summary>
-    /// The RegressionMetricsStatistics class is computes summary
-    /// statistics over multiple observations of regression evaluation metrics.
+    /// This interface handles the accumulation of summary statistics over multiple observations of
+    /// evaluation metrics.
     /// </summary>
-    public sealed class RegressionMetricsStatistics : MetricsStatisticsBase<RegressionMetrics>
+    /// <typeparam name="T">The metric results type, such as <see cref="RegressionMetrics"/>.</typeparam>
+    internal interface IMetricsStatistics<T>
+    {
+        void Add(T metrics);
+    }
+
+    /// <summary>
+    /// The <see cref="RegressionMetricsStatistics"/> class holds summary
+    /// statistics over multiple observations of <see cref="RegressionMetrics"/>.
+    /// </summary>
+    public sealed class RegressionMetricsStatistics : IMetricsStatistics<RegressionMetrics>
     {
         /// <summary>
         /// Summary Statistics for L1
```

Comment:
do the `summary` comments here need to be updated?

Reply:
I thought I'd opportunistically update them, since one was wrong and I think they all kind of benefit by describing (since it is no longer part of the type signature) what *sort* of thing they're accumulating. Do you have a concern here that I should not have updated them?

Reply:
I'm happy you changed the property names to match the underlying metrics class.

My concern is that the summary comments here are now out of date. What is `L1`? `L2`? etc.

Reply:
Oh, you meant not this line, but the summary comments on the individual statistics. Yes, I am doing that.